### PR TITLE
Log skipped prevoting as INFO

### PR DESCRIPTION
**Pull request by pxsalehi**
Relates ES-6576

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java [hunk 1]
```diff
@@ -1781,7 +1781,7 @@ public void run() {
                         final var nodeEligibility = localNodeMayWinElection(lastAcceptedState, electionStrategy);
                         if (nodeEligibility.mayWin() == false) {
                             assert nodeEligibility.reason().isEmpty() == false;
-                            logger.trace(
+                            logger.info(
                                 "skip prevoting as local node may not win election ({}): {}",
```
Comment (DiannaHohensee) [inline server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java:1785]: You'll need to follow up with a serverless test fix to testShutdownLogMessageDuringElectionAttempt, which expects a TRACE log (also activates TRACE logging via @TestLogging for that check)

**Review by DiannaHohensee — APPROVED** [review]
Could you elaborate on
1. why we're opting for INFO now instead of TRACE ([this was the previous discussion, where we were concerned about overly chatty logging](https://github.com/elastic/elasticsearch/pull/106701#discussion_r1542659541))
2. would we want to also change [the other log](https://github.com/elastic/elasticsearch/blob/616e71963e195ed3306fb2721c139f6477b33e8f/server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java#L551-L555) message?

The change is fine to me, I'm not familiar with us having much experience with the msg to have a preference either way at the moment, but I would like to close the loop on a couple things, if we can.

File: server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java (hunk 1 above)
Comment (DiannaHohensee) [reply server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java:1785]: Ha, I just saw your PR for this. Nvm 🙂

**Comment by pxsalehi** [comment]
> why we're opting for INFO now instead of TRACE (https://github.com/elastic/elasticsearch/pull/106701#discussion_r1542659541)

I think the intention of ES-6576 was to be able to see these in logs when checking some deployments logs. I think the number of these should be limited to only when the node has a shutdown marker. Otherwise, we'd have to change the log level and wait until we see new logs, rather than being able to spot these in the logs right away. Also I've been told `TRACE` logs don't even get ingested! The linked discussion seems to be for another log which is staying `TRACE` here. In any case, as you mentioned, we can change these as we find out how useful/noisy they are.

> would we want to also change [the other log](https://github.com/elastic/elasticsearch/blob/616e71963e195ed3306fb2721c139f6477b33e8f/server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java#L551-L555) message?

If I'm not mistaken, that seems to be (at least in stateless) what is used in `preVoteCollector.start` and should be covered in the shutdown scenario we are testing by the log changed in this PR. So I'm not sure we want that one or not. I've just changed the one that we have a test for.

**Comment by pxsalehi** [comment]
@elasticmachine update branch

**Pull request merged**